### DISCUSSION TITLE
Copying one by one from the tray

### DISCUSSION THREAD

**Discussion by Hela-Girl** [hluk/CopyQ#2161] [Q&A]
The program has a tray in which the entries are in order 1, 2, 3, 4 .... X how can I assign a hot key to each entry? For example on the 1st record I want to set ctrl+1. On the 2nd entry, ctrl+2, and so on. Is it possible?

**Comment by hluk** [comment]
By default 1..9 keys would trigger the tray menu items - unless "Search for numbers" (in History config tab) is enabled, or you open the menu by clicking on the tray icon and desktop environment overrides the default menu.

**Reply by Hela-Girl** [reply]
I'm not looking for. but without calling the program window, configure the hotkeys to record 1, 2, 3, 4....X. To avoid calling up the window and choosing with the mouse, and just press the shortcut keys and the required information from the clipboard is inserted into the input field at once.

Do you understand what I mean? ))

**Reply by hluk** [reply]
Maybe this is what you are looking for: https://github.com/hluk/copyq-commands/tree/master/Global#paste-nth-item ([here is how to add the command to CopyQ](http://copyq.readthedocs.io/en/latest/faq.html#how-to-load-shared-commands-and-share-them))

**Reply by Hela-Girl** [reply]
Yeah, that's what we needed!!!! **Thank you!**
